Update array functions for Statamic 5 / Laravel 11

dchymko opened the pull request:
Fixes `function array_get() undefined` on the second step of the importer when attempting to import the posts. 

Is it worth including logic to keep it compatible with < 5.0?

Also, I believe @kevmarsden has some additional fixes that may supersede this one, in which case, this can be closed.

kevmarsden commented:
This PR works as expected as long as the `config/statamic-wp-import.php` file is renamed to `wp-import.php`.  

There are still some issues with taxonomy imports, so I'll create a separate PR with those fixes once these pending PRs are merged.

ryanmitchell commented:
@kevmarsden I've reverted the config changes, looked like a find/replace error to me

dchymko commented:
> @kevmarsden I've reverted the config changes, looked like a find/replace error to me

I changed the references, because with the plugin namespace/rename change to statamaic-rad-pack today, it was throwing an `no hint path defined for` error. 

If you reinstall the add-on with composer, you should be able to reproduce.

jackmcdade commented:
Thank you!
